plugins/modules/packaging/os/pkgng.py

```diff
@@ -23,14 +23,20 @@
     name:
         description:
             - Name or list of names of packages to install/remove.
-        required: true
+        required: false
     state:
         description:
             - State of the package.
             - 'Note: "latest" added in 2.7'
         choices: [ 'present', 'latest', 'absent' ]
         required: false
         default: present
+    upgrade:
+        description:
+            - Upgrade all installed packages.
+        type: bool
+        required: false
+        default: no
```

Comment:
```suggestion
        default: no
        version_added: 1.0.0
```

Reply:
Hmm, the module isn't really versioned right now AFAICT... do I need to add that as well?

Reply:
The collection is versioned, not the module. And the next collection release will be 1.0.0, so this option will show up in community.general 1.0.0.

Reply:
Understood, thanks for the clarification. Rebased and applied just now.